src/main/resources/db/migration/V10__trim_user_info.sql

```diff
@@ -0,0 +1,10 @@
+ALTER TABLE user_info
+    DROP COLUMN id_number,
+    DROP COLUMN date_of_birth,
+    DROP COLUMN place_of_birth,
+    DROP COLUMN document_type,
+    DROP COLUMN document_number,
+    DROP COLUMN document_country,
+    DROP COLUMN document_valid_from,
+    DROP COLUMN document_valid_until,
+    DROP COLUMN deactivated;
```

Comment:
not sure about this. Waiting for @Ivan Kardum to provide more info on the user data that we have to store. It looks like we will have to store following:
> First/Last Name
> Residence
> Date of birth
> Personal identification number

We'll decide today.

Reply:
Who is Ivan Kardum